.eslintrc

```diff
@@ -1,5 +1,11 @@
 {
   "extends": ["@nuxt/eslint-config"],
+  "ignorePatterns":[
```

Comment:
Please split it to another `chore:` PR 🙏🏼

Reply:
Sorry, reverted this commit: 73ae4eacdf70b900708bef38de1b44c7820d2c97

This content would like to requested for review in a separate PR.